libs/blocks/marquee/marquee.js

```diff
@@ -3,7 +3,8 @@
  */
 
 import { decorateButtons, getBlockSize, decorateBlockBg } from '../../utils/decorate.js';
-import { createTag, getConfig, loadStyle } from '../../utils/utils.js';
+import { createTag } from '../../utils/utils.js';
+import { decorateMnemonicList } from '../mnemonic-list/mnemonic-list.js';
```

Comment:
if `mnemonic-list` is not needed for all marquees on adobe.com, it should be imported dynamically.

Reply:
Addressed. It is now dynamically imported.